### ISSUE TITLE
-config flag listed as optional, is actually required

### ISSUE THREAD

**Issue by zeushammer** [Comcast/eel#31]
`./eelsys -h` produces
```Usage of ./eelsys:
  -config string
    	path to config.json (optional)
```

however

`./eelsys` produces
```
{ "error" : "open config-eel/config.json: no such file or directory" }{"cause":"open_config","error":"open config-eel/config.json: no such file or directory","error_type":"get_config","log.id":"","log.level":"error","log.timestamp":"2017-01-18T03:44:02.080403207Z"}
```

but

`./eelsys -config ../../config-eel/config.json`

works correctly

**Comment by boriwo** [comment]
By default eel looks for config.json in the config-eel folder, if the config file is not in that place (maybe because ur binary is located elsehwere) then u need to use the -config parameter.

**Comment by zeushammer** [comment]
Will be resolved in [#38](https://github.com/Comcast/eel/pull/38)
